.github/workflows/docs.yml

```
@@ -0,0 +1,38 @@
+name: docs-deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+
+      - name: deps
+        run: make dev
+
+      - name: docs
+        run: |
+          make doc
+          mv ./html/pip_audit /tmp/pip_audit
+
+      - name: deploy
+        run: |
+          # Configure git, checkout a new orphan branch for the docs.
+          git config --global user.name 'woodruffw'
+          git config --global user.email 'woodruffw@users.noreply.github.com'
```

```suggestion
          git config --global user.name 'github.actions'
          git config --global user.email 'github-actions@github.com'
```
ref: https://github.com/actions/checkout#push-a-commit-using-the-built-in-token